app/src/main/java/org/simple/clinic/drugs/selectionv2/entry/CustomPrescriptionEntrySheetv2.kt

```diff
@@ -44,7 +45,13 @@ class CustomPrescriptionEntrySheetv2 : BottomSheetActivity() {
     setContentView(R.layout.sheet_custom_prescription_entry_v2)
     TheActivity.component.inject(this)
 
-    Observable.mergeArray(sheetCreates(), drugNameChanges(), drugDosageChanges(), drugDosageFocusChanges(), saveClicks())
+    Observable.mergeArray(
+        sheetCreates(),
+        drugNameChanges(),
+        drugDosageChanges(),
+        drugDosageFocusChanges(),
+        saveClicks(),
+        removeClicks())
         .observeOn(io())
```

Comment:
The arguments to `mergeArray()` and operators on the Observable are on the same indentation. Can you instead try moving `mergeArray()` to a next line?

```kotlin
Observable
    .mergeArray(
        sheetCreates(),
        drugNameChanges(),
        drugDosageChanges(),
        drugDosageFocusChanges(),
        saveClicks(),
        removeClicks())
    .observeOn(io())
    .compose(controller)
    .observeOn(mainThread())
    .takeUntil(onDestroys)
    .subscribe { uiChange -> uiChange(this) }
```